undebt/pattern/interface.py

```diff
@@ -78,10 +84,23 @@ def patterns_from_files(pattern_files):
 
 def load_module(path):
     """Loads a module from its path."""
+    if module_like(path):
+        path = module_name_to_path(path)
```

Comment:
@ajm188 Yup, that's generally the way it's done. Although you want to prepend it, and probably use `os.getcwd()` instead of `"."`.